### ISSUE TITLE
Hello teacher, I have the following question and would like to ask how you can solve it.

### ISSUE THREAD

**Issue by Ysy0219** [abman23/seq2seq-SC#3]
Gradients do not exist for variables,when minimizing the loss. If you're using `model.compile()`, did you forget to provide a `loss` argument?
